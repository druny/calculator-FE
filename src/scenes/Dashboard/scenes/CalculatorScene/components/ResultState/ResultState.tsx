import React from 'react';

type ResultStateProps = {
  result: string;
};

export function ResultState({ result }: ResultStateProps) {
  return (
    <div>
      <p>{result || 'Hey?!!!'}</p>
    </div>
  );
}
