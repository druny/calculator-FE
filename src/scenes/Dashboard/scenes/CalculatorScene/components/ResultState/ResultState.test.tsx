import React from 'react';
import renderer from 'react-test-renderer';

import { ResultState } from './ResultState';

it('renders correctly when there are no items', () => {
  const tree = renderer.create(<ResultState result="1" />).toJSON();
  expect(tree).toMatchSnapshot();
});
