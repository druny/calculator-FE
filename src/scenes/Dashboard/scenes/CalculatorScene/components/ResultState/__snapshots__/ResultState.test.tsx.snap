// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<div>
  <p>
    1
  </p>
</div>
`;
